src/webapp/src/utilities/reporting.ts

```diff
@@ -0,0 +1,48 @@
+import * as uuid from 'uuid'
+
+import Reporter, { HumbugConsent } from '@bugout/humbug'
+
+const HUMBUG_TOKEN = 'a4bb7056-94b9-4857-893e-dc8e4fa41345'
+
+const packagePath = process.cwd()
+const packageConfigFile = require(`${packagePath}/package.json`)
+export let tags: string[] = [packageConfigFile.version]
+
+
+type reportingConfigType = {
+    clientId: string
+    consent: boolean
+}
+
+
+const getReportingConfig = (): reportingConfigType => {
+    let reportingConfig: reportingConfigType
+    const fs = require('fs')
+    try {
+        const reportingConfigRaw = fs.readFileSync(`${packagePath}/reporting_config.json`, 'utf8')
+        const reportingConfigJson = JSON.parse(reportingConfigRaw)
+        reportingConfig = { clientId: reportingConfigJson.clientId, consent: reportingConfigJson.consent }
+    } catch {
+        reportingConfig = { clientId: uuid.v4(), consent: true }
+        fs.writeFile(`${packagePath}/reporting_config.json`, JSON.stringify(reportingConfig), (err: Error) => {
+            if (err) {
+                console.log("Unable to create reporting_config.json file")
+            }
+        })
+    }
+
+    return reportingConfig
+}
+
+const sessionId = uuid.v4()
+const loadedReportingConfig = getReportingConfig()
```

Comment:
Instead of `reporting_config.json`, use environment variables. This will be deployed as a web service and environment variables are the best practice compared to config files (harder to create config files as part of automatic deployment).

Reply:
I think this will also make your code simpler.